client/src/components/FeedbackPage/FeedbackFormSection/FeedbackFormSection.spec.tsx

```diff
@@ -0,0 +1,157 @@
+      render(<FeedbackFormSection />);
+      const radioButtons = getRadioButtons(0);
+      expect(radioButtons.length).toEqual(5);
+      radioButtons.forEach(radioButton => checkIfNotCheckedButRequired(radioButton));
+    });
+
+    it('should render 2 radio buttons labelled "Yes" and "No" respectively and they should not be checked but required', () => {
+      render(<FeedbackFormSection />);
+      const radioButtons = getRadioButtons(1);
+      expect(radioButtons.length).toEqual(2);
+      radioButtons.forEach(radioButton => checkIfNotCheckedButRequired(radioButton));
+      expect(radioButtons[0]).toHaveAccessibleName('Yes');
+      expect(radioButtons[1]).toHaveAccessibleName('No');
+    });
+
+    it('should render a button labelled "Submit a form"', () => {
+      render(<FeedbackFormSection />);
+      expect(screen.getByRole('button', { name: 'Submit a form' })).toBeInTheDocument();
+    });
+
+    it('should render a blank form', () => {
+      render(<FeedbackFormSection />);
+      expect(screen.getByRole('form')).toHaveFormValues(initialFormValues);
+    });
+  });
+
+  describe('when typed in a textbox field', () => {
+    it('should not render errors if typed value is valid', () => {
+      render(<FeedbackFormSection />);
+      const textboxElements = screen.getAllByRole('textbox');
+      const errorMessageElements = screen.getAllByRole('alert');
+      textboxElements.forEach(textboxElement => {
+        fireEvent.change(textboxElement, { target: { value: 'Some value' } });
+        errorMessageElements.forEach(errorMessageElement => {
+          expect(errorMessageElement.textContent).toEqual('');
+        });
+      });
+    });
+  });
+
+  describe('when a submit event is fired on the form', () => {
+    it('should call dispatch function and reset form to a blank', async () => {
+      render(<FeedbackFormSection />);
+      const formElement = screen.getByRole('form');
+      const textboxElements = screen.getAllByRole('textbox');
+      const starRating1Button = getRadioButtons(0)[0];
+      const yesButton = getRadioButtons(1)[0];
+
+      textboxElements.forEach(textboxElement =>
+        fireEvent.change(textboxElement, { target: { value: 'Some value' } })
+      );
+      fireEvent.click(starRating1Button);
+      fireEvent.click(yesButton);
+      expect(formElement).toHaveFormValues({
+        name: 'Some value',
+        rating: '1',
+        reason: 'Some value',
+        suggestion: 'Some value',
+        recommendation: 'Yes',
+        more: 'Some value'
+      });
+
+      fireEvent.submit(formElement);
+      expect(mockDispatch).toHaveBeenCalledTimes(1);
+      expect(mockDispatch).toHaveBeenCalledWith(
+        addFeedback({
+          name: 'Some value',
+          rating: Number('1'),
+          reason: 'Some value',
+          suggestion: 'Some value',
+          recommendation: 'Yes',
+          more: 'Some value',
+          timestamp: new Date(1482363367071)
```

Comment:
have you tried just to execute test with `new Date()` only? i think it might be passed

Reply:
yes, please commit with update)